src/main/java/duke/Command.java

```diff
@@ -0,0 +1,54 @@
+package duke;
+
+import java.util.HashMap;
+
+/**
+ * Represents commands entered by the input.
+ * Commands are to be processed by Parser.
+ */
+public enum Command {
```

Comment:
Wow nice use of enum with HashMap to convert String to Command. Didn't think of that. 👍